libraries/chain/controller.cpp

```diff
@@ -3757,10 +3772,10 @@ void controller::start_block( block_timestamp_type when,
                     bs, std::optional<block_id_type>(), deadline );
 }
 
-void controller::finalize_block( block_report& br, const signer_callback_type& signer_callback ) {
+void controller::finish_block( block_report& br, const signer_callback_type& signer_callback ) {
```

Comment:
I think I would have called it `complete_block` as it switches to the `completed_block` stage.

Reply:
@arhag mentioned calling it `finish_block` so just went with that. I'm ok with either `finish_block` or `complete_block`.

Reply:
I think `finish_block` highlights the last step of those block building stages. Sounds more natural.